frontend-zs/src/components/projects/Home/ProjectCard.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Row, Card, Col } from 'react-bootstrap'
+import { Redirect } from 'react-router-dom'
+
+function goToProject(project) {
+    return <Redirect to={'/projects' + project._id} />
+
+}
+
+function ProjectCard(project, index) {
+    return (
+        <Row key={index}>
+            <Card bg="dark" text="white" className="text-left mt-md-3 mr-md-2 ml-md-2" tag="a"
```

Comment:
I would propose to use their design guidline for the background color of the card. Maybe the dark grey color: #5F696A